api/envoy/extensions/filters/http/aws_request_signing/v3/aws_request_signing.proto

```diff
@@ -39,6 +39,10 @@ message AwsRequestSigning {
   // Example: s3
   string service_name = 1 [(validate.rules).string = {min_len: 1}];
 
+  // Optional region string. If region is not provided, the region will be retrieved from the environment
+  // or AWS configuration files. The region string is mandatory if signing_algorithm is set to ``AWS_SIGV4A`` and will
+  // be checked at runtime.
```

Comment:
What will be preferred between ENV and configuration file?

Reply:
ok - there will be some slight behaviour changes as I've received confirmation from the service team as to how this can be configured environment/config-wise. i also documented the behaviour in https://github.com/envoyproxy/envoy/pull/32070/files#diff-f75c1708c2e1316d0d66a9b1af39bf0ca022c116ea97f6403826372be3699a53 , however i will xref it here to make it very clear.

Reply:
oh, I see. Thanks, the docs is very clear. Then I think you can add a link to the docs in the proto field comments.

Reply:
should be in there now :)